Cell.xctemplate/___FILEBASENAME___CellModel.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ___FILENAME___
+//  ___PROJECTNAME___
+//
+//  Created by ___FULLUSERNAME___ on ___DATE___.
+//  Copyright © ___YEAR___ ___ORGANIZATIONNAME___. All rights reserved.
+//
+
+import UIKit
+
+class ___FILEBASENAMEASIDENTIFIER___CellModel {
+
+    init() {
+
+    }
+}
+
+extension ___FILEBASENAMEASIDENTIFIER___CellModel: CellModel {
+    func cellType() -> UIView.Type {
+        return ___FILEBASENAMEASIDENTIFIER___Cell.self
+    }
+
+    var cellHeight: CGFloat {
```

Comment:
Tohle má v sobě už implementovaný protokol, tak je otázka jestli to tady na sílu nastavovat na 100.

Reply:
ja myslim ze by sme v tej sablone urcite uz mali mat cellHeight pretoze ho v 90% pripadoch pouzivame (a to asi nechceme zakazdym vypisovat) tym padom tam bolo treba dat nejaku hodnotu

Reply:
Mi se prostě nelíbí ta 100 – to číslo nemá žádný podklad, ani to není standardní výška buněk, to bych raději všude dal automatickou dimenzi.